index.qmd

```diff
@@ -4,8 +4,19 @@ title: "R-multiverse"
 
 <img src="logo/logo-hexbin.png" align="right" width="140" />
 
-A community-curated collection of R package releases, powered by [R-universe](https://r-universe.dev), available for installation from the repository at <https://community.r-multiverse.org>.
+#### A dual package repository system, powered by R-universe
 
-Anyone can contribute packages to the GitHub repository at <https://github.com/r-multiverse/contributions>. Once a package is registered, every new release is automatically built and checked with results available at <https://community.r-multiverse.org>. Package maintainers retain complete control over deployment.
+#### [Community](community.qmd)
 
-The R-multiverse project originated from the [R Consortium Repositories Working Group](https://github.com/RConsortium/r-repositories-wg) and is supported by an [R Consortium](https://www.r-consortium.org/) ISC grant.
+An open and inclusive R package repository with transparent governance standards, built on open source infrastructure.
+
+#### +
```

Comment:
?

Reply:
Think this is just the GitHub diff causing confusion, this text is still retained later.